litmus-portal/backend/graphql-server/pkg/usermanagement/create.go

```diff
@@ -0,0 +1,47 @@
+package usermanagement
```

Comment:
We are having all the mutations, queries and subscription helper functions in the graphql package.